linkerd/app/core/src/errors.rs

```diff
@@ -1,150 +1,177 @@
-//! Layer to map HTTP service errors into appropriate `http::Response`s.
-
-use crate::svc;
-use futures::{Future, Poll};
-use http::{header, Request, Response, StatusCode, Version};
+use crate::proxy::{buffer, identity};
+use http::{header::HeaderValue, StatusCode};
 use linkerd2_error::Error;
+use linkerd2_error_respond as respond;
 use linkerd2_proxy_http::HasH2Reason;
-use tracing::{debug, error, warn};
-
-/// Layer to map HTTP service errors into appropriate `http::Response`s.
-pub fn layer() -> Layer {
-    Layer
-}
-
-#[derive(Clone, Debug)]
-pub struct Layer;
+use linkerd2_router::error as router;
+use tower::load_shed::error as shed;
+use tower_grpc::{self as grpc, Code};
+use tracing::debug;
```

Comment:
Teensy style nit: I notice we import `debug!` by name, but refer to `warn!` as `tracing::warn!`...is this just because there are multiple `debug!`s and only a single `warn!`?